test/unit-test/mqtt_agent_utest.c

```diff
@@ -93,6 +93,11 @@ static uint8_t packetType;
  */
 static uint16_t packetIdentifier;
 
+/**
+ * @brief Return flags to use for test.
+ */
+static MQTTAgentCommandFuncReturns_t pReturnFlags;
```

Comment:
Should this be memset to 0 for every test?

Reply:
It is not a pointer. I have changed the name.

Reply:
It not being a pointer doesn't affect whether we should reset it or not. If we use it for multiple tests we should probably reset it.